scalalib/src/mill/javalib/spotless/KotlinSpotlessModule.scala

```diff
@@ -0,0 +1,35 @@
+package mill.javalib.spotless
+
+import mill.scalalib.DepSyntax
+import mill.{Agg, PathRef, T, Task}
+
+trait KotlinSpotlessModule extends SpotlessModule {
+
+  /**
+   * Classpath for running
+   * - Ktfmt
+   * - ktlint
+   */
+  def jvmLangLibClasspath: T[Seq[PathRef]] = Task {
+    defaultResolver().resolveDeps(
+      Agg(
+        ivy"com.facebook:ktfmt:${ktfmtVersion()}",
+        ivy"com.pinterest.ktlint:ktlint-cli:${ktlintVersion()}"
+      )
+    )
+  }
+
+  /**
+   * Defaults to `0.53`.
+   */
+  def ktfmtVersion: T[String] = Task {
+    "0.53"
```

Comment:
Should not be hardcoded. This version is already managed. 

https://github.com/com-lihaoyi/mill/blob/facbe11cf4d2ab176aa16780ddcf509e67d5a2cd/kotlinlib/src/mill/kotlinlib/ktfmt/KtfmtModule.scala#L26